web/src/compose_recipient.ts

```diff
@@ -83,12 +85,31 @@ export function update_narrow_to_recipient_visibility(): void {
             compose_state.has_full_recipient()
         ) {
             $(".conversation-arrow").toggleClass("narrow_to_compose_recipients", true);
+            show_tippy_tooltip();
             return;
         }
     }
     $(".conversation-arrow").toggleClass("narrow_to_compose_recipients", false);
 }
 
+function show_tippy_tooltip(): void {
+    const arrow_element = document.querySelector(".conversation-arrow");
+
+    if (arrow_element) {
+        const instance = tippy(arrow_element, {
+            content: "Go to conversation",
+            trigger: "manual",
+            placement: "top",
+        });
```

Comment:
Instead of creating a new tippy instance every time `show_tippy_tooltip` is called, you could check if an instance already exists on the element and reuse it. This avoids creating duplicate tooltips and improves performance.

```suggestion
        const instance = arrow_element._tippy || tippy(arrow_element, {
            content: "Go to conversation",
            trigger: "manual",
            placement: "top",
        });
```